providers/openfeature-provider-flagd/tests/e2e/config.feature

```diff
@@ -0,0 +1,87 @@
+Feature: Configuration Test
```

Comment:
I know this is a copy of the other test, but generally, I do think it might be worth adding this test to the harness to ensure all providers have the same config and support the same values. (even if they might not need this config currently)

Reply:
Maybe also adapting it, to be more feature specific

Reply:
https://github.com/open-feature/flagd-testbed/pull/156